volestipy/tests/bigg_files/e_coli_core.json

```diff
@@ -0,0 +1,14786 @@
+{
```

Comment:
this is extremely huge for inclusion in the library, if it is not publicly available I recommend to create another repository or a dataset (e.g. in zenodo) and put it there

Reply:
EDIT: it is available here: http://bigg.ucsd.edu/models/e_coli_core so you have to retrieve it from there

Reply:
Files removed from the repo.